contracts/token/ERC20/extensions/ERC4626.sol

```diff
@@ -97,77 +97,77 @@ abstract contract ERC4626 is ERC20, IERC4626 {
     }
 
     /**
+     * @inheritdoc IERC20Metadata
+     *
      * @dev Decimals are computed by adding the decimal offset on top of the underlying asset's decimals. This
      * "original" value is cached during construction of the vault contract. If this read operation fails (e.g., the
      * asset has not been created yet), a default of 18 is used to represent the underlying asset's decimals.
```

Comment:
This doesn't work in the docs. It just takes the next `@dev` tag. I would keep the old NatSpec here instead.

```suggestion
     * @inheritdoc IERC20Metadata
     *
     * @dev Decimals are computed by adding the decimal offset on top of the underlying asset's decimals. This
     * "original" value is cached during construction of the vault contract. If this read operation fails (e.g., the
     * asset has not been created yet), a default of 18 is used to represent the underlying asset's decimals.
     *
     * See {IERC20Metadata-decimals}.
```